enclave/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_library(oecryptombed STATIC
+    ../../common/asn1.c
+    ../../common/cert.c
+    ../../common/kdf.c
+    asn1.c
+    cert.c
+    crl.c
+    ec.c
+    cmac.c
+    hmac.c
+    key.c
+    random.c
+    rsa.c
+    sha.c)
+
+maybe_build_using_clangw(oecryptombed)
+
+target_include_directories(oecryptombed PRIVATE
+    ${PROJECT_SOURCE_DIR}/include/openenclave/corelibc)
+
+target_link_libraries(oecryptombed PUBLIC
+    oe_includes
+    mbedcrypto)
```

Comment:
So there will be 4 crypto libraries? liboecryptombed.a, libmbedtls.a, ….
Is there a plan to merge all of them to a single library?

Reply:
mbedtls is actually 3 libraries (mbedtls, libmbedx509, libmbedcrypto). Since we're saying that you can use the crypto library we use for your enclave applications, I don't think we should combine them, since users would expect those three mbedtls libraries to exist.